File: cronografo/Models/ShotModel.cs
using System;

namespace cronografo.Models
{
    public class ShotModel
    {
        public int Number { get; set; }
        public float Velocity { get; set; }
        public float Energy { get; set; }
        public DateTime Timestamp { get; set; } = DateTime.Now;

        public string DisplayTime => Timestamp.ToString("HH:mm:ss");
        public string DisplayVelocity => $"{Velocity:F1} m/s";
        public string DisplayEnergy => $"{Energy:F1} J";
    }
}

File: cronografo_android/app/src/main/java/com/luis/cronografo/ShotModel.kt
package com.luis.cronografo

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

data class ShotModel(
    val number: Int,
    val velocity: Float,
    val energy: Float,
    val timestamp: Long = System.currentTimeMillis()
) {
    val displayTime: String
        get() {
            val sdf = SimpleDateFormat("HH:mm:ss", Locale.getDefault())
            return sdf.format(Date(timestamp))
        }
        
    val displayVelocity: String
        get() = String.format(Locale.US, "%.1f m/s", velocity)
        
    val displayEnergy: String
        get() = String.format(Locale.US, "%.1f J", energy)
}
